### PR TITLE
add new fields to vsc stations && update tests files

### PR DESCRIPTION
**Pull request by bendaamerahm**
- add new fields to vsc stations

- update tests files

### CODE REVIEW

File: src/test/resources/all-map-data-in-variant.json [hunk 1]
```diff
@@ -483,15 +483,21 @@
...
       "terminalConnected": true,
       "hvdcLineId": null,
-      "lossFactor": 1.0
+      "lossFactor": 1.0,
+      "voltageRegulatorOn": true,
+      "voltageSetpoint": 150.0,
+      "reactivePowerSetpoint": 40.0
     }
   ]
 }
```
Comment (sBouzols) [inline src/test/resources/all-map-data-in-variant.json:503]: add line at end of file

File: src/test/resources/all-map-data.json [hunk 2]
```diff
@@ -471,15 +471,21 @@
...
       "terminalConnected": true,
       "hvdcLineId": null,
-      "lossFactor": 1.0
+      "lossFactor": 1.0,
+      "voltageRegulatorOn": true,
+      "voltageSetpoint": 150.0,
+      "reactivePowerSetpoint": 40.0
     }
   ]
 }
```
Comment (sBouzols) [inline src/test/resources/all-map-data.json:491]: add line at end of file

File: src/test/resources/partial-all-map-data-in-variant.json [hunk 3]
```diff
@@ -92,7 +92,10 @@
...
       "terminalConnected": true,
       "hvdcLineId": null,
-      "lossFactor": 1.0
+      "lossFactor": 1.0,
+      "voltageRegulatorOn": true,
+      "voltageSetpoint": 150.0,
+      "reactivePowerSetpoint": 40.0
     }
   ]
 }
```
Comment (sBouzols) [inline src/test/resources/partial-all-map-data-in-variant.json:101]: add line at end of file

File: src/test/resources/partial-all-map-data.json [hunk 4]
```diff
@@ -80,7 +80,10 @@
...
       "terminalConnected": true,
       "hvdcLineId": null,
-      "lossFactor": 1.0
+      "lossFactor": 1.0,
+      "voltageRegulatorOn": true,
+      "voltageSetpoint": 150.0,
+      "reactivePowerSetpoint": 40.0
     }
   ]
 }
```
Comment (sBouzols) [inline src/test/resources/partial-all-map-data.json:89]: add line at end of file

**Review by JacquesBorsenberger — APPROVED** [review]

**Review by sBouzols — APPROVED** [review]
Code review OK
Tests OK
console warnings checked OK

**Pull request merged**